src/portal/views/Dao.vue

```diff
@@ -1,199 +1,215 @@
+          </v-toolbar>
+          <v-tabs-items v-model="tabSelected">
+            <v-tab-item v-for="(tab, i) in tabs" :key="i" :value="`tab-${i}`">
+              <v-container>
+                <v-text-field
+                  v-model="searchTerm"
+                  color="primary darken-2"
+                  label="Search by proposal action or description"
+                ></v-text-field>
 
-                  <div class="d-flex align-center text-center threshold">
-                    <v-divider vertical />
-                    <span
-                      >Threshold: <br />{{ proposal.nayes.length + proposal.ayes.length }}/{{ proposal.threshold }}
-                    </span>
-                    <v-divider vertical />
-                  </div>
-                  <v-btn
-                    color="grey lighten-2 black--text"
-                    @click="openVoteDialog(proposal.hash, false)"
-                    :width="`${(proposal.nayes.length / (proposal.ayes.length + proposal.nayes.length)) * 100 + 100}`"
-                    :disabled="loadingVote"
-                    >No <v-divider class="mx-3" vertical />{{ proposal.nayes.length }}
-                  </v-btn>
-                </v-container>
-                <v-container v-if="proposal.ayesProgress > 0 || proposal.nayesProgress > 0">
-                  <v-row justify="center">
-                    <v-progress-linear
-                      :value="proposal.ayesProgress"
-                      height="25"
-                      :style="{
-                        width: proposal.ayesProgress + '%',
-                        marginRight: 'auto',
-                        backgroundColor: '#1982b1',
-                        color: '#fff',
-                      }"
-                    >
-                      <template>
-                        <strong
-                          >{{
-                            !!(proposal.ayesProgress % 1) ? proposal.ayesProgress.toFixed(2) : proposal.ayesProgress
-                          }}%</strong
+                <v-card class="my-3 pa-3" outlined v-for="(proposal, i) in filteredProposals(tab.content)" :key="i">
+                  <v-card-title class="pa-3 mb-2">
+                    {{ proposal.action.toUpperCase() }}
+                  </v-card-title>
+                  <v-card-subtitle class="pb-0">
+                    Proposal hash:
+                    <div class="pl-2">
+                      <p class="text--secondary">{{ proposal.hash }}</p>
+                    </div>
+                  </v-card-subtitle>
+                  <v-card-text class="pb-0">
+                    Description:
+                    <div class="pl-2">
+                      <p class="text--secondary">
+                        {{ proposal.description }}
+                        <a v-bind:href="proposal.link" v-bind:target="'blank'">More details</a>
+                      </p>
+                    </div>
+                    <p v-if="proposal.end > Date.now()">
```

Comment:
Can we change the style for the proposal hash & description to be like this?

Reply:
on the same line, you mean?